static/js/components/forms/ProfileFormFields.js

```diff
@@ -108,10 +108,16 @@ export const LegalAddressFields = ({
       <Field
         type="text"
         name="legal_address.first_name"
+        id="legal_address.first_name"
         className="form-control"
         autoComplete="given-name"
+        aria-describedby="legal_address.first_nameError"
```

Comment:
Our original mistake here was naming these form fields in a way that matches the API results, instead of giving them camel case names like our other forms and mapping the API results to those fields. This field should have `name="firstName"` and the error should have `id="firstNameError"`. That issue existed long before this PR though, so you definitely don't need to change all of these values here. In the meantime, this mixture of snake case and camel case is awkward. Can you change the naming scheme so that this value is `legal_address.first_name.error` or `legal_address.first_name_error`, whichever one you prefer? After that, please apply the same naming scheme to these other attributes that were added in this PR.